extension/js/common/composer.ts

```diff
@@ -1201,6 +1208,45 @@ export class Composer {
         if (method === 'forward') {
           this.urlParams.subject = 'Fwd: ' + this.urlParams.subject;
           await this.retrieveDecryptAddForwardedMsg(determined.lastMsgId);
+        } else {
+          try {
+            const { raw } = await Google.gmail.msgGet(this.urlParams.acctEmail, this.urlParams.threadMsgId, 'raw');
```

Comment:
Although `threadMsgId` usually points to the last message, this is not guaranteed and I'd like to stop passing it into compose one day.

You can use `threadId` to pull a list of messages, then grab the last one. Please look at what forward is doing, and re-use (some of) the code.

Reply:
Okay, I will use threadId to get a list and then pick up the last message from the list